po/ta.po

```diff
@@ -0,0 +1,7195 @@
+"Language: ta_IN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: geany.desktop.in:5 data/geany.glade:6456
+msgid "Geany"
+msgstr "சீனி"
+
+#: geany.desktop.in:6
+msgid "Integrated Development Environment"
+msgstr "ஒருங்கிணைந்த மேம்பாட்டு சூழல்"
+
+#: geany.desktop.in:7
+msgid "A fast and lightweight IDE using GTK+"
+msgstr "GTK+ ஐப் பயன்படுத்தி வேகமான மற்றும் இலகுரக IDE"
+
+#: geany.desktop.in:14
+msgid "Text;Editor;"
+msgstr "Editor"
+
+#: data/geany.glade:130
+msgid "_Toolbar Preferences"
+msgstr "_Toolbar விருப்பத்தேர்வுகள்"
+
+#: data/geany.glade:147
+msgid "_Hide Toolbar"
+msgstr "_ஹைட் கருவிப்பட்டி"
+
+#: data/geany.glade:305 data/geany.glade:6714
+msgid "_Edit"
+msgstr "_தொகு"
+
+#: data/geany.glade:313 data/geany.glade:6929
+msgid "_Format"
+msgstr "_FORMAT"
+
+#: data/geany.glade:319
+msgid "I_nsert"
+msgstr "செருகவும்"
+
+#: data/geany.glade:332 data/geany.glade:7082
+msgid "Insert _ChangeLog Entry"
+msgstr "_Changelog நுழைவைச் செருகவும்"
+
+#: data/geany.glade:341 data/geany.glade:7091
+msgid "Insert _Function Description"
+msgstr "செயல்பாட்டு விளக்கத்தை செருகவும்"
+
+#: data/geany.glade:350 data/geany.glade:7100
+msgid "Insert Mu_ltiline Comment"
+msgstr "மல்டிலைன் கருத்தை செருகவும்"
+
+#: data/geany.glade:357 data/geany.glade:7367
+msgid "_More"
+msgstr "_ மோர்"
+
+#: data/geany.glade:370 data/geany.glade:7115
+msgid "Insert File _Header"
+msgstr "கோப்பு _ஹெடரைச் செருகவும்"
+
+#: data/geany.glade:379 data/geany.glade:7124
+msgid "Insert _GPL Notice"
+msgstr "_GPL அறிவிப்பை செருகவும்"
+
+#: data/geany.glade:388 data/geany.glade:7133
+msgid "Insert _BSD License Notice"
+msgstr "_BSD உரிம அறிவிப்பை செருகவும்"
+
+#: data/geany.glade:405 data/geany.glade:7144
+msgid "Insert Dat_e"
+msgstr "Dat_e செருகவும்"
+
+#: data/geany.glade:417 data/geany.glade:439 data/geany.glade:6602
+#: data/geany.glade:7049 data/geany.glade:7156 data/geany.glade:7178
+#: data/geany.glade:7885 data/geany.glade:7905
+msgid "invisible"
+msgstr "கண்ணுக்கு தெரியாத"
+
+#: data/geany.glade:427 data/geany.glade:7166
+msgid "_Insert \"include <...>\""
+msgstr "_Insert \"அடங்கும் <...>\""
+
+#: data/geany.glade:457 data/geany.glade:7190 src/keybindings.c:516
+msgid "Insert Alternative _White Space"
+msgstr "மாற்று _white இடத்தை செருகவும்"
+
+#: data/geany.glade:470 data/geany.glade:7231
+msgid "_Search"
+msgstr "_ தேடல்"
+
+#: data/geany.glade:482 data/geany.glade:6526
+msgid "Open Selected F_ile"
+msgstr "தேர்ந்தெடுக்கப்பட்ட f_ile ஐத் திறக்கவும்"
+
+#: data/geany.glade:493 data/geany.glade:7400 src/symbols.c:2205
+msgid "Find _Usage"
+msgstr "_USAGE ஐக் கண்டறியவும்"
+
+#: data/geany.glade:504 data/geany.glade:7409 src/symbols.c:2210
+msgid "Find _Document Usage"
+msgstr "_Document பயன்பாட்டைக் கண்டறியவும்"
+
+#: data/geany.glade:515 data/geany.glade:7439
+msgid "Go to Symbol Defini_tion"
+msgstr "Go to Tag Definition"
```

Comment:
Same here?  I'm not gonna do the whole file though, and if you're happy with the translation that's fine by me and I'll ley @frlan deal with this in the end anyway :wink: